APITests/Utilities/RatingUtilsTests.cs

```diff
@@ -110,4 +110,152 @@ public void ProvisionalFlag_IsFalse_WhenNotProvisional(double volatility, int ma
+		Assert.Equal(expectedSubTier, subTier);
+	}
+	
+	[Theory]
+	[InlineData(RatingUtils.RatingBronzeIII + 5, RatingUtils.RatingBronzeII-(RatingUtils.RatingBronzeIII+5))]
+	[InlineData(RatingUtils.RatingBronzeII + 5, RatingUtils.RatingBronzeI-(RatingUtils.RatingBronzeII+5))]
+	[InlineData(RatingUtils.RatingBronzeI + 5, RatingUtils.RatingSilverIII-(RatingUtils.RatingBronzeI+5))]
+	[InlineData(RatingUtils.RatingSilverIII + 5, RatingUtils.RatingSilverII-(RatingUtils.RatingSilverIII+5))]
+	[InlineData(RatingUtils.RatingSilverII + 5, RatingUtils.RatingSilverI-(RatingUtils.RatingSilverII+5))]
+	[InlineData(RatingUtils.RatingSilverI + 5, RatingUtils.RatingGoldIII-(RatingUtils.RatingSilverI+5))]
+	[InlineData(RatingUtils.RatingGoldIII + 5, RatingUtils.RatingGoldII-(RatingUtils.RatingGoldIII+5))]
+	[InlineData(RatingUtils.RatingGoldII + 5, RatingUtils.RatingGoldI-(RatingUtils.RatingGoldII+5))]
+	[InlineData(RatingUtils.RatingGoldI + 5, RatingUtils.RatingPlatinumIII-(RatingUtils.RatingGoldI+5))]
+	[InlineData(RatingUtils.RatingPlatinumIII + 5, RatingUtils.RatingPlatinumII-(RatingUtils.RatingPlatinumIII+5))]
+	[InlineData(RatingUtils.RatingPlatinumII + 5, RatingUtils.RatingPlatinumI-(RatingUtils.RatingPlatinumII+5))]
+	[InlineData(RatingUtils.RatingPlatinumI + 5, RatingUtils.RatingEmeraldIII-(RatingUtils.RatingPlatinumI+5))]
+	[InlineData(RatingUtils.RatingEmeraldIII + 5, RatingUtils.RatingEmeraldII-(RatingUtils.RatingEmeraldIII+5))]
+	[InlineData(RatingUtils.RatingEmeraldII + 5, RatingUtils.RatingEmeraldI-(RatingUtils.RatingEmeraldII+5))]
+	[InlineData(RatingUtils.RatingEmeraldI + 5, RatingUtils.RatingDiamondIII-(RatingUtils.RatingEmeraldI+5))]
+	[InlineData(RatingUtils.RatingDiamondIII + 5, RatingUtils.RatingDiamondII-(RatingUtils.RatingDiamondIII+5))]
+	[InlineData(RatingUtils.RatingDiamondII + 5, RatingUtils.RatingDiamondI-(RatingUtils.RatingDiamondII+5))]
+	[InlineData(RatingUtils.RatingDiamondI + 5, RatingUtils.RatingMasterIII-(RatingUtils.RatingDiamondI+5))]
+	[InlineData(RatingUtils.RatingMasterIII + 5, RatingUtils.RatingMasterII-(RatingUtils.RatingMasterIII+5))]
+	[InlineData(RatingUtils.RatingMasterII + 5, RatingUtils.RatingMasterI-(RatingUtils.RatingMasterII+5))]
+	[InlineData(RatingUtils.RatingMasterI + 5, RatingUtils.RatingGrandmasterIII-(RatingUtils.RatingMasterI+5))]
+	[InlineData(RatingUtils.RatingGrandmasterIII + 5, RatingUtils.RatingGrandmasterII-(RatingUtils.RatingGrandmasterIII+5))]
+	[InlineData(RatingUtils.RatingGrandmasterII + 5, RatingUtils.RatingGrandmasterI-(RatingUtils.RatingGrandmasterII+5))]
+	[InlineData(RatingUtils.RatingGrandmasterI + 5, RatingUtils.RatingEliteGrandmaster-(RatingUtils.RatingGrandmasterI+5))]
+	[InlineData(RatingUtils.RatingEliteGrandmaster + 5, 0)]
+	public void GetRatingDelta_ReturnsRatingDelta_GivenCurrentRating(double rating, double expectedDelta)
+	{
+		// Arrange
+		
+		// Act
+		double delta = RatingUtils.GetRatingDeltaForNextTier(rating);
+		
+		// Assert
+		Assert.Equal(expectedDelta, delta);
+	}
```

Comment:
Here I would utilize different parameters because these `InlineData`s look a bit confusing. i.e. `(double rating, double offset)`

I'd even recommend using `RatingUtils.GetNextTier` (or some similar method) to get the next tier, then apply the delta in the `// Act` portion.

Also, just declaring variables is something that goes in the `// Arrange` section, `// Act` is for like calculations or other logic. So I would request this be addressed in the rest of your tests.